oak/client/application_client.h

```diff
@@ -147,6 +147,48 @@ class ApplicationClient {
   }
 };
 
+// Convert a grpc::StatusCode into human-readable string.
+std::string status_code_to_string(grpc::StatusCode code) {
```

Comment:
Is this really not defined anywhere else?

Reply:
https://github.com/grpc/grpc/issues/17969 was closed for inactivity.